cas-server-webapp/src/main/webapp/WEB-INF/spring-configuration/applicationContext.xml

```diff
@@ -68,4 +68,25 @@
             value-ref="serviceTicketUniqueIdGenerator" />
     </util:map>
 
+    <bean id="passThroughController" class="org.springframework.web.servlet.mvc.UrlFilenameViewController"/>
```

Comment:
Only the root webapp context is scanned. Not the cas-servlet files. So this bean could not have been available for context manipulation if it was in the servlet context. I could move the scan down month servlet context but that was not only so much more work, it also created lots of other issues. 

It makes no difference really. All endpoints are functional.